src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -72,17 +46,19 @@
  * <h3>How Mutect2 works compared to HaplotypeCaller</h3>
  * <p>Overall, Mutect2 works similarly to HaplotypeCaller, but with a few key differences. </p>
  * <p>
- *     (i) GVCF calling is not a feature of Mutect2.
- *     (ii) While HaplotypeCaller relies on a fixed ploidy assumption to inform its genotype likelihoods that are the basis for genotype probabilities (PL),
- *     Mutect2 allows for varying ploidy in the form of allele fractions for each variant.
+ *     <dl>
+ *     <dd>(i) GVCF calling is not a feature of Mutect2.</dd>
+ *     <dd>(ii) While HaplotypeCaller relies on a fixed ploidy assumption to inform its genotype likelihoods that are the basis for genotype probabilities (PL),
+ *     Mutect2 allows for varying ploidy in the form of allele fractions for each variant.</dd>
  *     Varying allele fractions is often seen within a tumor sample due to fractional purity, multiple subclones and/or copy number variation.
- *     (iii) Mutect2 also differs from the HaplotypeCaller in that it can apply various prefilters to sites and variants depending on the use of
+ *     <dd>(iii) Mutect2 also differs from the HaplotypeCaller in that it can apply various prefilters to sites and variants depending on the use of
  *     a matched normal (--normalSampleName), a panel of normals (PoN; --normal_panel) and/or a common population variant resource containing allele-specific frequencies (--germline_resource).
- *     If provided, Mutect2 uses the PoN to filter sites and the germline resource and matched normal to filter alleles.
- *     (iv) Mutect2's default variant site annotations differ from those of HaplotypeCaller. See the --annotation parameter description for a list.
- *     (v) Finally, Mutect2 has additional parameters not available to HaplotypeCaller that factor in the decision to reassemble a genomic region,
+ *     If provided, Mutect2 uses the PoN to filter sites and the germline resource and matched normal to filter alleles.</dd>
+ *     <dd>(iv) Mutect2's default variant site annotations differ from those of HaplotypeCaller. See the --annotation parameter description for a list.</dd>
+ *     <dd>(v) Finally, Mutect2 has additional parameters not available to HaplotypeCaller that factor in the decision to reassemble a genomic region,
  *     factor in likelihood calculations that then determine whether to emit a variant, or factor towards filtering.
- *     These parameters include the following and are each described further in the arguments section.
+ *     These parameters include the following and are each described further in the arguments section.</dd>
+ *     </dl>
```

Comment:
Is the content here still germane? For example, the `--min_variants_in_pileup`, is it still used in active region determination, etc?

Reply:
Fyi,https://gatkforums.broadinstitute.org/dsde/discussion/10911/differences-between-gatk3-mutect2-gatk4-mutect2-and-haplotypecaller-placeholder/p1?new=1 and https://docs.google.com/a/broadinstitute.org/document/d/1pV5eQOAGl88wBt5HrhAsWME-ygv-6MeC8DTm-S2OJus/edit?usp=sharing

Reply:
I deleted a lot.  Also, I kebabified all of the examples.